### DISCUSSION TITLE
Mistaken processing or onboarding of forked repos in hosted Renovate app

### DISCUSSION THREAD

**Discussion by driimus** [renovatebot/renovate#21243] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

None

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

I've recently noticed that the hosted version of Renovate bot has started processing my repository forks, despite it not being configured to do so. 

Some examples:
- driimus/fastify-type-provider-typebox#1 - repo without a `renovate` config
- driimus/prisma#68 - repo with `.github/renovate.json`. I've had to disable most repo features to reduce noise in the meantime.
- driimus/renovate-test-fork-processing#1 - repo with `renovate.json`. I've forked your repo specifically to test whether it's still happening. 

Based on the documentation for [forkProcessing](https://docs.renovatebot.com/configuration-options/#forkprocessing) (which has replaced in #20759), I would expect automatic fork processing to only happen if I've only enabled the app for a specific set of repositories, which is not the case:

![image](https://user-images.githubusercontent.com/32523430/228637256-c3479c55-401c-42a3-a268-0096d48f0cc0.png)

Without having looked too much into the issue, I'd be interested to know whether there's a straightforward way to mitigate this as an end-user.

### Relevant debug logs

<details><summary>Logs</summary>

No logs are available in the app's dashboard for [affected repositories](https://github.com/driimus?tab=repositories&q=&type=fork&language=&sort=). Example:
![image](https://user-images.githubusercontent.com/32523430/228639083-958e1bf1-1df1-4f2a-8162-f34c89222a4b.png)

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by HonkingGoose** [comment]
I got a PR from Renovate bot on my **fork** of `renovatebot/renovate` 5 days ago:

- https://github.com/HonkingGoose/renovate/pull/6

I'm also allowing Renovate to run on all repositories, and am also using the Mend Renovate hosted app on github.com

I think we have a valid bug report here, and I'm labeling it high priority.

**Comment by rarkins** [comment]
Are you still seeing this same behavior since ~5 days ago?

It's unfortunately not a bug in Renovate, but instead an unfortunate mistake in our backend related to changes to fork processing in v35. It runs repos individually, which meant forkProcessing=enabled got implicitly turned on.

We should have stopped processing forked repos like this, but a cleanup job remains.

**Comment by HonkingGoose** [comment]
> Are you still seeing this same behavior since ~5 days ago?

I'm not seeing this on my fork of the `renovatebot/renovate` repository anymore. 😄 

---

> We should have stopped processing forked repos like this, but a cleanup job remains.

Is this why some of the "bad" PRs from Renovate remain open, like these two from the original post?

> - https://github.com/driimus/fastify-type-provider-typebox/pull/1
> - https://github.com/driimus/renovate-test-fork-processing/pull/1

**Comment by rarkins** [comment, marked as answer]
I have converted this to a discussion, as this was a mistake in our app backend and not a bug directly of Renovate. We will repair/recover this hopefully within the next week via implementing #21242 and turning it on in the week long enough to clean up the mistaken PRs
